Services/Worker/OJS.Services.Worker.Business/ExecutionContext/Implementations/ExecutionContextBuilderService.cs

```diff
@@ -50,6 +50,8 @@ public OjsSubmission<TInput> BuildOjsSubmission<TInput>(SubmissionServiceModel s
                 string.Format(CannotCreateInputTemplate, submissionServiceModel.ExecutionType))
             .VerifyResult();
 
+        submission.StartedExecutionOn = DateTime.UtcNow;
```

Comment:
`BuildOjsSubmission` should be included in the calculation. It is part of the execution. Move as close as possible to the entry point. Ideally when the request enters.

Reply:
Done.

Reply:
Can you put it before the validation logic also? Should be the first thing in the service

Reply:
Done.

Reply:
I men to move the point where StartedExecutionOn is assigned the DateTime.UtcNow value. We want to start measuring this time as soon as the submission hits the worker.

Reply:
Done.